Kickstarter-iOS/Views/Cells/ProjectPamphletMainCell.swift

```diff
@@ -49,8 +49,6 @@ internal final class ProjectPamphletMainCell: UITableViewCell, ValueCell {
   @IBOutlet fileprivate weak var readMoreButton: UIButton!
   @IBOutlet fileprivate weak var stateLabel: UILabel!
   @IBOutlet fileprivate weak var statsStackView: UIStackView!
-  @IBOutlet fileprivate weak var youreABackerContainerView: UIView!
```

Comment:
How come we removed this? 🤔

Reply:
Product decision - It's not in the design and it would be redundant to have `You're a backer` label on the project cell and manage CTA.